js/utilities/UniqueIdService.js

```diff
@@ -40,6 +40,37 @@ class UniqueIdService {
                 element.setAttribute('id', hash);
             });
     }
+
+    /**
+     * Update selectWoo child elements with unique IDs
+     * @param group {HTMLElement}
+     */
+    uniquifySelectWoo (group) {
+        const orginalSelect = group.querySelector('select.js-select2');
+        const orginalSelectId = orginalSelect.getAttribute('id');
+
+        [].slice.call(group.querySelectorAll('.select2-container'))
+            .forEach(select2Container => {
+                if (select2Container.querySelector('.select2-selection--multiple')) {
```

Comment:
Can we be explicit on these if statements in this method, not essential in this scenario, but it always feels a gamble to me to hand off the expression return value to the "truthy" / "falsy" gods

Seems more robust to stick to expressions that return boolean values (but it might just be me!)

```if (select2Container.querySelector('.select2-selection--multiple') !== null) {```